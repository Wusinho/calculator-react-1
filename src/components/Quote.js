import React from 'react';

const Quote = () => (
  <>
    <h3>Quotes of the day:</h3>
    <p>
      There is no good and bad
    </p>
    <p>
      It is all a matter of taste
    </p>
  </>
);

export default Quote;
