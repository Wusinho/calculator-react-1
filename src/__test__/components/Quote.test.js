import React from 'react';
import { create } from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import Quote from '../../components/Quote';

it('Should render Quote page', () => {
  const elem = create(<Quote />).toJSON();
  expect(elem).toMatchSnapshot();
});
it('Should render quotes', () => {
  render(<Quote />);
  const elem = screen.getByText('There is no good and bad');
  expect(elem).toBeInTheDocument();
});
