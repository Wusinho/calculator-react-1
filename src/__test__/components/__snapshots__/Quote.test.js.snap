// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Quote page 1`] = `
Array [
  <h3>
    Quotes of the day:
  </h3>,
  <p>
    There is no good and bad
  </p>,
  <p>
    It is all a matter of taste
  </p>,
]
`;
